.spellcheck-en-custom.txt

```diff
@@ -1,124 +1,125 @@
+AMDGPU
 Abhishek
 Akash
-AMDGPU
-arge
-arXiv
-backend
-backends
-backport
-backported
 Bhandwaldar
-cli
 CLI
+CRB
+CUDA
 Colab
-compositional
 Conda
 Containerfile
-coreutils
-cpp
-CRB
-cuBLAS
-CUDA
-dataset
-dev
-ditaa
 DKMS
-dr
 Dropdown
-env
 EPEL
 Eval
 Finetuning
-Gaudi
 GFX
 GGUF
+Gaudi
 GiB
 GitHub
 Gmail
-gpu
-habana
-Habana
-hipBLAS
 HPU
-ilab
-impactful
-Inferencing
-instructlab
+Habana
 ISA
+Inferencing
 JIT
-journald
 Jupyter
-Kaggle
 KAGGLE
+Kaggle
 Kaggle's
 Kai
 Kubernetes
-lignment
 LLM
-llms
 LLVM
-lora
+MLX
+MPS
 Merlinite
 Miniforge
 Mixtral
-mlx
-MLX
-MPS
-Nvidia
 NVidia
 NVidia
+Nvidia
 OCI
-oneMKL
-orchestrator
-ots
-Pareja
 PEFT
+Pareja
 PlantUML
 Podman
-pre
-preprint
-pyenv
 PyPI
 PyTorch
-qlora
-quantized
 Quantizing
-Radeon
 RDNA
 README
-repo
 RHEL
 ROCm
 RTX
-runtime
 RX
-Salawu
+Radeon
 SDG
-sexualized
 SHA
+Salawu
 Shaders
 Shivchander
 Srivastava
-subdirectory
-submodules
 Sudalairaj
-Taj
-tatsu
 TBD
+Taj
 Tesla
-th
-tl
-tox
 UI
-unquantized
 USM
-venv
-wikisql
 WSL
-xcode
 XNACK
 XT
 XTX
 Xu
 YAML
 Zsh
+arXiv
+arge
+backend
+backends
+backport
+backported
+cli
+compositional
+coreutils
+cpp
+cuBLAS
+dataset
+dev
+ditaa
+dr
+env
+gpu
+habana
+hipBLAS
+ilab
+impactful
+instructlab
+journald
+lignment
+llms
+lora
+mlx
+oneMKL
+orchestrator
+ots
+pre
+preprint
+pyenv
+qlora
+quantized
+repo
+runtime
+sexualized
+subdirectory
+submodules
+tatsu
+th
+tl
+tox
+triagers
+unquantized
+venv
+wikisql
+xcode
```

Comment:
Spellcheck sort I think needs to be run?

Reply:
funny enough, these changes are the result of running `make spellcheck-sort`. I only added one word.